x-pack/plugins/security_solution/server/lib/risk_engine/retrieve_risk_inputs.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ElasticsearchClient } from '@kbn/core/server';
+import type { QueryDslBoolQuery } from '@elastic/elasticsearch/lib/api/types';
+import { groupBy } from 'lodash';
+import type { RiskInput, SimpleRiskInput } from './types';
+
+const buildBoolQueryForIdsOfIndex = (index: string, ids: string[]): QueryDslBoolQuery => ({
+  must: { terms: { _id: ids } },
+  filter: { term: { _index: index } },
+});
+
+export const retrieveRiskInputs = async ({
```

Comment:
Yeah, if you look in the commit history and find where this was added, you'll see I had [described the situation](https://github.com/elastic/kibana/pull/155966/commits/26cafcdb445e028bd2fb09d25ea81432759cced8). 

TL;DR this function is not currently used, but I intend it will be leveraged to support the Risk Inputs aspect of risk explainability: given a risk score, return the alerts that contributed to it. However, I will keep this file locally and use it when needed instead of committing it in this PR.